app/main.py

```diff
@@ -1,3 +1,15 @@
 def cache(func):
-    # Write your code here
-    pass
+    data = {}
+
+    def inner(*args, power: int = 0):
```

Comment:
variable "power" doesn't really do anything. try rewrite your code without it.

Reply:
thank you, i wrote this variable for test myself this decorator and forget to clean

Reply:
it's classic one) you are welcome!